startup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bundle install
+
+PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.publishing.service.gov.uk} \
+bundle exec rackup spec/dummy/config.ru --port=3212 --host=0.0.0.0
```

Comment:
We can remove this as running the app is addressed in https://github.com/alphagov/govuk_publishing_components/pull/115